v23.1/tsquery.md

```diff
@@ -0,0 +1,63 @@
+---
+title: TSQUERY
+summary: The TSQUERY data type a list of lexemes separated by operators, and is used in full-text search.
+toc: true
+docs_area: reference.sql
+---
+
+The `TSQUERY` [data type](data-types.html) stores a list of lexemes separated by operators. `TSQUERY` values are used in [full-text search](full-text-search.html).
+
+## Syntax
+
+A `TSQUERY` comprises individual lexemes and operators in the form: `'These' & 'lexemes' & 'are' & 'not' & 'normalized.'`.
+
+The operators in a `TSQUERY` are used to [match a `TSQUERY` to a `TSVECTOR`](full-text-search.html#match-queries-to-documents). Valid `TSQUERY` operators are:
+
+- `&` (AND)
+- `|` (OR)
+- `!` (NOT)
+- `<->` (FOLLOWED BY)
```

Comment:
Do we want to add any more detail on what these mean? Additionally, the followed by operator has another parameter - you can do `<->` which is equivalent to `<1>`, but you can also do `<n>` like `<3>` or `<5>` which asks for the things on the left and right to be separated by n other lexemes.

Reply:
Added examples.